<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html xmlns="http://www.w3.org/1999/xhtml"><head>	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />	<meta name="generator" content="ScreenSteps http://www.screensteps.com/" />	<title>FASTX-Toolkit</title>	<link href="../DEhelp.css" media="screen" rel="stylesheet" type="text/css" charset="utf-8" /></head><body class="trebuchet">	<div id="wrapper">				<div id="LessonContent">			<div class="LessonHeader">			  <p><a href="../Discovery_Environment_Manual.html">Back to Table of Contents</a></p>				<h1 class="LessonTitle">FASTX-Toolkit</h1>			</div>										<div id="step_1" class="lessonStep top">	    		 <p>The FASTX-Toolkit is a collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing. Next-Generation sequencing machines usually produce FASTA or FASTQ files, containing multiple short-reads sequences (possibly with quality information).</p><p>The main processing of such FASTA/FASTQ files is mapping (aka aligning) the sequences to reference genomes or other databases using specialized programs. Example of such mapping programs are: Blat, SHRiMP, LASTZ, MAQ and many many others.</p><p>However, it is sometimes more productive to preprocess the FASTA/FASTQ files before mapping the sequences to the genome - manipulating the sequences to produce better mapping results. The FASTX-Toolkit tools perform some of these preprocessing tasks.</p><p><font color="#000000">Above description from the FASTX-Toolkit website: </font><a href="http://hannonlab.cshl.edu/fastx_toolkit/" target="_blank">http://hannonlab.cshl.edu/fastx_toolkit/</a></p><p>The following are currently enabled:<br /><font color="#000000">&bull; Barcode Splitter - Splits a FASTQ file containing multiple samples.</font><br /><font color="#000000">&bull; Clipper - Removes sequencing adapters / linkers from FASTQ files.</font><br /><font color="#000000">&bull; Groomer (Quality Rescaler) - Converts FASTQ files from Illumina 1.3+ and Solexa formats to Sanger PHRED format. This is not listed on the main </font>FASTX-Toolkit<font color="#000000"> page, but is a part of the suite. See: </font><a href="http://main.g2.bx.psu.edu/root?tool_id=fastq_groomer" target="_blank">http://main.g2.bx.psu.edu/root?tool_id=fastq_groomer</a><br /><font color="#000000">&bull; Quality Filter - Filters FASTQ formatted sequences based on quality.</font><br /><font color="#000000">&bull; Trimmer - Trims (cuts) barcodes or noise from FASTQ sequences.</font></p><p><font color="#000000">Author:</font><br /><font color="#000000">Hannon Lab at Cold Spring Harbor Laboratory: </font><a href="http://hannonlab.cshl.edu/" target="_blank">http://hannonlab.cshl.edu/</a></p><p>This tool was identified for inclusion by the iPlant Genotype to Phenotype working group. The 0.3.x release of the Discovery Environment uses FASTX-Toolkit version 0.0.13.</p>	</div>	<div class="clear"></div>		</div>		<div id="LessonPageNav">		  <p></p>		  <ul>		    <li id="LessonPagePrevious"><a href="Cufflinks.html">Previous Page: Cufflinks</a> &nbsp;</li>		    <li id="LessonPageCount">Page <strong>36</strong> of <strong>44</strong></li>		    <li id="LessonPageNext"><a href="R_Language_and_Environment.html">Next Page: R Language and Environment</a> &nbsp;</li>		    		  </ul>		</div>	</div>	</body></html>