<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html xmlns="http://www.w3.org/1999/xhtml"><head>	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />	<meta name="generator" content="ScreenSteps http://www.screensteps.com/" />	<title>Test Data (New)</title>	<link href="../DEhelp.css" media="screen" rel="stylesheet" type="text/css" charset="utf-8" /></head><body class="trebuchet">	<div id="wrapper">				<div id="LessonContent">			<div class="LessonHeader">			  <p><a href="../Discovery_Environment_Manual.sstcpkg">Back to Table of Contents</a></p>				<h1 class="LessonTitle">Test Data (New)</h1>			</div>										<div id="step_1" class="lessonStep top">	    		 <p>A quick description of each of the sample data files provided in the iPlant Discovery Environment. Data is organized based on the tool or analysis for which each is intended.</p>	</div>	<div class="clear"></div>	<div id="step_2" class="lessonStep top">	        <h2 class="StepTitle">FASTX</h2>		 <p><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;s_1_sequence.txt - This data set is comprised of a single lane of 8,296,093 36bp Illumina 1.2 single-end sequence reads.  This library is a composite of three barcoded sequence libraries: 1) Barcode3: CTCGT; 2) Barcode6: CGACT; and 3) Barcode9: TAGCT.</font></p><p><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;s_5_1_sequence.txt - This data set is comprised of a single lane of Illumina 1.3+ reads, sequenced in the pair-end context.  This file is run 1 and s_5_2_sequence.txt is run 2.  This library is a composite of three barcoded sequence libraries: 1) Barcode1: GATCT; 2) Barcode2: ATCGT; and 3) Barcode 12: ACAGT.  There are many sequences in this data that are of low quality, so the user will wish to filter them out before proceeding.  After filtering, the user will wish to reconcile the reads, such that only reads where both Run1 and Run2 records exist are propagated into downstream analysis. Finally, the user will wish to convert these sequence reads from PHRED64 (Illumina 1.3) format to PHRED33 (Sanger) quality scaling.</font></p><p><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;s_5_2_sequence.txt - This data set is comprised of a single lane of Illumina 1.3+ reads, sequenced in the pair-end context.  This file is run 2 and s_5_1_sequence.txt is run 1.  This library is a composite of three barcoded sequence libraries: 1) Barcode1: GATCT; 2) Barcode2: ATCGT; and 3) Barcode 12: ACAGT.  There are many sequences in this data that are of low quality, so the user will wish to filter them out before proceeding.  After filtering, the user will wish to reconcile the reads, such that only reads where both Run1 and Run2 records exist are propagated into downstream analysis. Finally, the user will wish to convert these sequence reads from PHRED64 (Illumina 1.3) format to PHRED33 (Sanger) quality scaling.</font></p><p><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;s_8_sequence.txt -This is a dataset comprised of 6632564 100 bp reads from <i>Arabidopsis thaliana</i> that were generated using a protocol that may result in a terminal 3' sequence adapter and is useful to learn and test our QC preprocessing (FASTX toolkit).  (CTGTAGGCACCATCAATCGTATGCCGTCTTCTGCTTG)</font></p>	</div>	<div class="clear"></div>	<div id="step_3" class="lessonStep top">	        <h2 class="StepTitle">Pipelines</h2>		 <p><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;VariantDetection</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;arabidopsis_thaliana</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;SRR013327.fastq -This is a dataset comprised of a single lane of 61638311 40bp Illumina reads from <i>Arabidopsis thaliana </i>Col-0.  This file can be imported from NCBI's Sequenced Read Archive (SRA).</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;Results</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;SRR013327.v10.vcf  -This file is the result of using BWA to align the SRR013327.fastq file to the <i>Arabidopsis thaliana</i> v10 genome and then using various utilities in SAMtools to process the alignment to determine the entire table of high-confidence SNPs.</font></p><p><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;oryza_sativa_japonica</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;DRR000022.fastq - This is a dataset comprised of a single lane of 20435886 32bp Illumina reads from <i>Oryza sativa japonica</i> variety Koshihikari.  This file can be imported from NCBI's Sequence Read Archive (SRA).</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;Results</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;DRR000022.v6.vcf -This file is the result of using BWA to align the DRR000022.fastq file to the <i>Oryza sativa japonica</i> v6 genome and then using various utilities in SAMtools to process the alignment to determine the entire table of high-confidence SNPs.</font></p><p><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;zea_mays</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;SRR026741.fastq -This is a dataset comprised of a single lane of 46980674 36bp Illumina reads from <i>Zea mays</i> inbred line Mo17.  This file can be imported from NCBI's Sequence Read Archive (SRA).</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;Results</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;SRR026741.v5a.vcf - This file is the result of using BWA to align the SRR026741.fastq file to the <i>Zea mays</i> v5a genome and then using various utilities in SAMtools to process the alignment to determine the entire table of high-confidence SNPs.</font></p>	</div>	<div class="clear"></div>	<div id="step_4" class="lessonStep top">	        <h2 class="StepTitle">SAMtools</h2>		 <p><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;FindSNPs</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;arabidopsis_thaliana</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;SRR013327.v10.sam -This SAM file is produced from a BWA alignment of SRR013327.fastq to the <i>Arabidopsis thaliana</i> v10 genome and can be used for variant detection.</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;Results</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;SRR013327.v10.vcf -This file is the result of using various utilities in SAMtools to process theSRR013327.v10.sam alignment to determine the entire table of high-confidence SNPs.</font></p><p><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;oryza_sativa_japonica</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;DRR000022.v6.sam --This SAM file is produced from a BWA alignment of DRR000022.fastq to the <i>Oryza sativa japonica </i>v6 genome and can be used for variant detection.</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;Results</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;DRR000022.v6.vcf -This file is the result of using various utilities in SAMtools to process the DRR000022.v6.sam alignment to determine the entire table of high-confidence SNPs.</font></p><p><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;zea_mays</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;SRR026741.v5a.sam -This SAM file is produced from a BWA alignment of SRR026741.fastq to the <i>Zea mays</i> v5a genome and can be used for variant detection.</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;Results</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;SRR026741.v5a.vcf -This file is the result of using various utilities in SAMtools to process the SRR026741.v5a.sam alignment to determine the entire table of high-confidence SNPs.</font></p>	</div>	<div class="clear"></div>	<div id="step_5" class="lessonStep top">	        <h2 class="StepTitle">Tuxedo</h2>		 <p><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;arabidopsis_thaliana</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;SRR013411.fastq -This is a dataset comprised of a 16391875 50bp reads from <i>Arabidopsis thaliana</i>.  This file can be imported from NCBI's Sequence Read Archive (SRA) and can be used as input to align RNA-Seq reads to genomes using the ultra high-throughput short read aligner </font><font color="#0000FF"><u>Bowtie</u></font><font color="#000000">, and then analyzed to identify splice junctions between exons.</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;Results</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;accepted_hits.v10.sam -This is a SAM file produced from aligning SRR013411.fastq to the <i>Arabidopsis thaliana</i> v10 reference genome using Bowtie powered by TopHat.</font></p><p><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;oryza_sativa_japonica</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;DRR000351.fastq - This is a dataset comprised of a 16391875 36bp reads from <i>Oryza sativa japonica</i>.  This file can be imported from NCBI's Sequence Read Archive (SRA) ) and can be used as input to align RNA-Seq reads to genomes using the ultra high-throughput short read aligner </font><font color="#0000FF"><u>Bowtie</u></font><font color="#000000">, and then analyzed to identify splice junctions between exons.</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;Results</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;accepted_hits.sam -This is a SAM file produced from aligning DRR000351.fastq to the <i>Oryza sativa japonica</i> v6 reference genome using Bowtie powered by TopHat.</font></p><p><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;zea_mays</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;SRR039512.fastq -This is a dataset comprised of a 9075105 36bp reads from <i>Zea mays</i>.  This file can be imported from NCBI's Sequence Read Archive (SRA) ) and can be used as input to align RNA-Seq reads to genomes using the ultra high-throughput short read aligner </font><font color="#0000FF"><u>Bowtie</u></font><font color="#000000">, and then analyzed to identify splice junctions between exons.</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;Results</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;accepted_hits.sam -This is a SAM file produced from aligning SRR039512.fastq to the <i>Zea mays</i> v5a reference genome using Bowtie powered by TopHat.</font></p>	</div>	<div class="clear"></div>	<div id="step_6" class="lessonStep top">	        <h2 class="StepTitle">TR</h2>		 <p><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;Dna.txt &ndash; A DNA sequence</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;geneID.txt &ndash; Gene and ID number</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;GOaccessions.txt &ndash; A list of common Gene Ontology numbers</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;GOterms.txt &ndash; A list of common Gene Ontology terms</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;Protein.txt &ndash; Some common protein sequences</font></p>	</div>	<div class="clear"></div>	<div id="step_7" class="lessonStep top">	        <h2 class="StepTitle">TNRS</h2>		 <p><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;test_file_1.csv - CVS list of names available to check against a database of canonical names to return to both exact and possible matches to the iPlant database.</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;test_file_1.txt &ndash; Taxamatch results for both exact and possible matches to the CVS name list.</font></p>	</div>	<div class="clear"></div>	<div id="step_8" class="lessonStep top">	        <h2 class="StepTitle">TE</h2>		 <p><font color="#000000">CACE</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;aesculus-traits-T.csv</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;aesculus-traits.csv &ndash; This file is the trait selected for analysis after the tree has been selected, the data is uploaded to the DE and the parameters are set.</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;aesculus-tree.txt &ndash; This file is the tree selected for analysis after data is uploaded to the DE and the parameters are set.</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;Results</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;In.LeafArea.cm2_nodes.txt &ndash; This file contains ancestral character estimates.</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;In.LeafArea.cm2_out.txt &ndash; This file is an ancestral character reconstruction as an estimate of the Brownian motion parameter and the log likelihood of the model.</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;In.LeafArea.cm2_plot.pdf &ndash; This is a visual plotting of the ancestral trait value estimates on the phylogenetic tree using a color gradient.</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;In.LeafArea.cm2_tree.txt &ndash; This file is a table of ancestral trait values and the corresponding 95% confidence intervals.</font></p><p><font color="#000000">Contrast</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;arctostaphylos-traits_2.txt</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;archtostaphylos-tree.txt &ndash; This file contains the tree data from which Contrast analyzes a tree.</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;Results</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;Outfile &ndash; File output by Contrast from reading a tree file that produces covariances, regressions, and correlations between characters for those contrasts.</font></p><p><font color="#000000">DACE</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;aquilegia-traits_T.csv &ndash;</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;aquilegia-traits.csv &ndash; This file contains the trait data that have been matched to a tree&rsquo;s data.</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;aquilegia-tree.txt &ndash; This file is the tree data that must have matching trait data</font><br /><font color="#000000">&bull;&nbsp;&nbsp;&nbsp;&nbsp;Results.</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;T1_nodes.txt &ndash; This file is an estimation of the transition rates and the proportional likelihoods of the ancestor&rsquo;s states.</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;T1_out.txt &ndash; This file the proportional likelihood of all possible states at the internal nodes.</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;Ti_plot.pdf &ndash; This file is a visual plotting on the phylogenetic tree using pie charts to represent the likelihoods for all possible states at the internal nodes</font><br /><font color="#000000">&nbsp;&nbsp;&nbsp;&nbsp;&bull;&nbsp;&nbsp;&nbsp;&nbsp;T1_tree.txt &ndash; This file is an estimate of the transition rate with its associated uncertainty and the log likelihood of the model.</font></p>	</div>	<div class="clear"></div>		</div>		<div id="LessonPageNav">		  <p></p>		  <ul>		    <li id="LessonPagePrevious"><a href="Viewer_and_Editor_Integration.html">Previous Page: Viewer and Editor Integration</a> &nbsp;</li>		    <li id="LessonPageCount">Page <strong>10</strong> of <strong>44</strong></li>		    <li id="LessonPageNext"><a href="Troubleshooting.html">Next Page: Troubleshooting</a> &nbsp;</li>		    		  </ul>		</div>	</div>	</body></html>